autoscheduler/scraper/models/course.py

```diff
@@ -1,17 +1,27 @@
 from django.db import models
 
-class Course(models.Model): 
+def generate_course_id(dept: str, course_num: str, term: str):
+    """ Generates the coures_id in the form of {dept}{course_num}-{term} """
+
+    return "".join((dept, course_num, '-', term))
+
+class Course(models.Model):
+    """ Basic course model """
+
     id = models.CharField(max_length=10, primary_key=True) # Could be dept+course_num
     dept = models.CharField(max_length=4, db_index=True) # CSCE
-    course_num = models.CharField(max_length=5, db_index=True) # i.e. 221
-    title = models.CharField(max_length=100) # Course title, i.e. "Data Structres & Algorithms"
+    course_num = models.CharField(max_length=5, db_index=True) # i.e. 314
+    title = models.CharField(max_length=100) # Course title, i.e. "Programming Languages"

```

Comment:
Data Structures & Algorithms was too long to fit on the same line, so changed to 314/Programming Languages